tutorials/classical_simulation/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>We will now run the AFM state preparation sequence from the Pulser tutorial with our noise models, and compare the results <em>with</em> the <em>noiseless </em>case.</p><p> </p><p>Does it make sense to have SPAM errors in this notebook ? </p><p>It does not seem that we implement SPAM errors in the end. I propose to delete the last line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>